action/configurable_action.go

```diff
@@ -0,0 +1,23 @@
+package action
+
+import (
+	"github.com/deislabs/cnab-go/driver"
+)
+
+type OperationConfigFunc func(op *driver.Operation) error
+
+// ConfigurableAction is used to define actions which can configure operations before they are executed.
+type ConfigurableAction struct {
+	// OperationConfig is an optional handler that applies additional
+	// configuration to an operation before it is executed.
+	OperationConfig OperationConfigFunc
```

Comment:
I would have prefer a slice here, so user can have multiple unitary config modifiers. I guess it would help factorizing between different actions. WDYT?